js/index.js

```diff
@@ -0,0 +1,60 @@
+
+var user_selection = prompt("What Calculator mode do you want to use? Basic, Advanced, BMI or Trip Calculator?");
+
```

Comment:
You include the Trip calculator as an option, but you haven't coded it in. Don't include things that aren't finished in your push to master - consider this your final product.